Source/WebCore/css/counterStyles.css

```diff
@@ -286,16 +286,15 @@
     /* ▸ */
 }
 
-/* cjk-earthly-branch and cjk-heavenly-stem are using 'alphabetic' system instead of 'fixed', as in the spec, for compatibility with legacy behavior and WPT tests (https://github.com/w3c/csswg-drafts/issues/8596). */
```

Comment:
Could you add a comment with the Issue link where you and the WG folks decided this was the right change?